scanners/ncrack/values.yaml

```diff
@@ -0,0 +1,15 @@
+parserImage:
+  registry: docker.io
+  repository: scbexperimental/parser-ncrack
+  tag: latest
+
+scannerJob:
+  resources: {}
+# scannerJob:
```

Comment:
Is there a reason these lines have to be commented out?

Reply:
This is done in all scanners and recommended by helm.
Basically don't impose resource restrictions by default but give users the ability to configure them on their own.